File: Algorithm/programmers/Level1/이상한 문자 만들기.py
def solution(s):
    answer = ''
    array = s.split(' ')
    count = 0
    for char in array:
        count += 1
        for i in range(len(char)):
            if i % 2 == 0:
                answer += char[i].upper()
            else:
                answer += char[i].lower()
        if count < len(array):
            answer += ' '

    return answer


def solution(s):
    answer = []

    for char in s.split(' '):
        word = ''
        for i in range(len(char)):
            c = char[i].upper() if i % 2 == 0 else char[i].lower()
            word += c
        answer.append(word)

    return ' '.join(answer)

File: Algorithm/programmers/Level1/이상한 문자 만들기.js
function solution(s) {
  let answer = [];
  let tmp = s.split(" ");

  for (let i = 0; i < tmp.length; i++) {
    let str = "";
    for (let j = 0; j < tmp[i].length; j++) {
      if (j % 2 === 0) str += tmp[i][j].toUpperCase();
      else str += tmp[i][j].toLowerCase();
    }
    answer.push(str);
  }

  return answer.join(" ");
}
